packages/plugin-flex/src/commands/flex/plugins/start.ts

```diff
@@ -175,6 +175,24 @@ export default class FlexPluginsStart extends FlexPlugin {
     }
   }
 
+  /**
+   * Throws an error if user inputted a taken port
+   * Returns the port if available
+   *
+   * @param port
+   * @returns
+   */
+  async getPort(): Promise<number> {
+    const port = await findPortAvailablePort('--port', this._flags.port || '3000');
```

Comment:
Actually, so sorry I just remembered, you can add a `default` to the `flags` argument for this `port` field. So then you can just do `this._flags.port` and Oclif will automatically add a default for it!